index.d.ts

```diff
@@ -0,0 +1,115 @@
+import { ActionContext, Module, Store } from 'vuex';
+import { OidcClientSettings, User, UserManager } from 'oidc-client';
+import { Route, RouteConfig } from 'vue-router';
+
+export interface VuexOidcClientSettings extends OidcClientSettings {
+  authority: string;
+  clientId: string;
+  redirectUri: string;
+  responseType: string;
+  scope: string;
+  maxAge?: string;
+  uiLocales?: string;
+  loginHint?: string;
+  acrValues?: string;
+  postLogoutRedirectUri?: string;
+  popupRedirectUri?: string;
+  silentRedirectUri?: string;
+}
+
+export interface VuexOidcStoreSettings {
+  namespaced?: boolean;
+  dispatchEventsOnWindow?: boolean;
+}
```

Comment:
isPublicRoute?: function;
routeBase?: string;
defaultSigninRedirectOptions?: object;
defaultSigninSilentOptions?: object;
defaultSigninPopupOptions?: object;

Reply:
These are more possible attributes for VuexOidcStoreSettings